formats/financial_release/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "campaign_page"
+  ],
+  "properties": {
+    "campaign_page": {
```

Comment:
Could this use the existing [`parent`](https://github.com/alphagov/govuk-content-schemas/blob/master/formats/base_links.json#L17-L21) links entry?